src/cpp/handler/handlerengine.cpp

```diff
@@ -1303,8 +1303,8 @@ class HandlerEngine::Private : public QObject
 	{
 		qRegisterMetaType<DetectRuleList>();
 
-		publishLimiter = new RateLimiter(this);
-		updateLimiter = new RateLimiter(this);
+		publishLimiter = new RateLimiter;
+		updateLimiter = new RateLimiter;
```

Comment:
these will now need to be cleaned up some other way

Reply:
thanks for catching this. working on a fix now